automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/AsyncImportSpec.scala

```diff
@@ -19,15 +20,22 @@ import org.scalatest.time.{Minutes, Seconds, Span}
 import org.scalatest.{FreeSpec, Matchers}
 import spray.json._
 
+import java.util.UUID
 import java.util.concurrent.TimeUnit
 import scala.io.Source
 import scala.util.{Failure, Success, Try}
 
-class AsyncImportSpec extends FreeSpec with Matchers with Eventually with ScalaFutures
-  with BillingFixtures with WorkspaceFixtures with Orchestration {
+class AsyncImportSpec
+  extends FreeSpec
+    with Matchers
+    with Eventually
+    with ScalaFutures
+    with WorkspaceFixtures
+    with Orchestration {
 
   val owner: Credentials = UserPool.chooseProjectOwner
   val ownerAuthToken: AuthToken = owner.makeAuthToken()
+  val billingAccountName: String = ServiceTestConfig.Projects.billingAccountId
```

Comment:
nitpick for consistency:
```suggestion
  val billingAccountId: String = ServiceTestConfig.Projects.billingAccountId
```
(and the usages of this val)

Reply:
(and in the other files)

Reply:
Thanks